doc/releases/release-notes-3.1.rst

```diff
@@ -768,6 +768,18 @@ Trusted Firmware-m
 Documentation
 *************
 
+* Reorganised and consolidated documentation for improved readability and
```

Comment:
@stephanosio can you remove the dots at the end of the sentences? We don't usually add those

Reply:
Euh, scratch that. Seems it's variable :)

Reply:
Yea, it seems everyone is doing their own thing in the release notes :) It would be good to establish and document the style guidelines in the future.